staging/src/k8s.io/kubectl/pkg/cmd/events/events.go

```diff
@@ -229,6 +229,7 @@ func (o *EventsOptions) Run() error {
 	if o.forName != "" {
 		listOptions.FieldSelector = fields.AndSelectors(
 			fields.OneTermEqualSelector("involvedObject.kind", o.forGVK.Kind),
+			fields.OneTermEqualSelector("involvedObject.apiVersion", o.forGVK.Group+"/"+o.forGVK.Version),
```

Comment:
```suggestion
			fields.OneTermEqualSelector("involvedObject.apiVersion", o.forGVK.GroupVersion().String()),
```
I think, CI job failures should be fixed with this suggested change.